jcl/src/java.base/share/classes/java/lang/StackTraceElement.java

```diff
@@ -260,6 +260,8 @@ void appendTo(Appendable buf) {
 	/*[IF Sidecar19-SE]*/
 	if (null != moduleName) {
 		appendTo(buf, moduleName);
+		appendTo(buf, "@"); //$NON-NLS-1$
```

Comment:
I don't think this is correct. Seems that printStackTrace() doesn't include the version, while the ThreadInfo printing does.

Reply:
In the reference implementation `StackTraceElement.toString()`  does not print the version or monitor information, nor does `Thread.dumpStack()`.  It appears that only `ThreadInfo.toString()` prints this extra information.